### PR TITLE
feat: added typography and lists

### PR DESCRIPTION
**Pull request by AnjaliDeshwani**
- Typography
  * typography.html
  * typography.css 
  *  Preview link- https://deploy-preview-6--winsome-ui.netlify.app/components/typography/typography

- Lists
  * lists.html
  * lists.css
  *  Preview link- https://deploy-preview-6--winsome-ui.netlify.app/components/lists/lists

### CODE REVIEW

File: components/Lists/lists.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+            Alert
+          </a>
+          <a href="/components/Badge/badge.html" class="nav-link-item">
+            Badge
+          </a>
+          <a href="/components/Button/button.html" class="nav-link-item">
+            Button
+          </a>
+          <a href="/components/Card/card.html" class="nav-link-item">Card</a>
+          <a href="/components/Image/image.html" class="nav-link-item active">
```
Comment (Tarandeep-s1ngh) [inline components/Lists/lists.html]: Give the "active" class to lists rather than image component

File: components/Typography/typography.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+            Alert
+          </a>
+          <a href="/components/Badge/badge.html" class="nav-link-item">
+            Badge
+          </a>
+          <a href="/components/Button/button.html" class="nav-link-item">
+            Button
+          </a>
+          <a href="/components/Card/card.html" class="nav-link-item">Card</a>
+          <a href="/components/Image/image.html" class="nav-link-item active">
```
Comment (Tarandeep-s1ngh) [inline components/Typography/typography.html]: Give the "active" class to Typography rather than the image component

**Review by Tarandeep-s1ngh — APPROVED** [review]

File: components/Typography/typography.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+.h5,
+h5 {
+  font-size: 0.5rem;
+  line-height: 1rem;
+}
+
+.font-black {
+  font-weight: 900;
+}
```
Comment (manismk) [inline components/Typography/typography.css]: .font-bolder would be better than .font-black

File: components/Typography/typography.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
+.h1,
+h1 {
```
Comment (manismk) [inline components/Typography/typography.css]: You can remove the direct tag for styling.

**Review by manismk — APPROVED** [review]

**Review by ayush65 — APPROVED** [review]

File: components/Lists/lists.html (hunk 1 above)
Comment (AnjaliDeshwani) [reply components/Lists/lists.html]: Thanks Tarandeep for pointing it out, I somehow missed it. I have updated it now.

File: components/Typography/typography.css (hunk 3 above)
Comment (AnjaliDeshwani) [reply components/Typography/typography.css]: Thanks Manikandan for suggesting these changes. I have pushed these changes.

**Pull request merged**